source/admin_manual/list_of_events.rst

```diff
@@ -854,579 +482,384 @@ These events apply only for connections using the ``connection API``.
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from environment (e.g. :ref:`event_mail_user`)                    |
-+---------------------+------------------------------------------------------+
+   .. todo:: WHAT DOES THIS MEAN???
 
-.. _event_fs_file:
 
-fs_file
--------
+.. dovecot_core:event:: fs_file
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from :ref:`event_fs` or any other specified event                 |
-| (e.g. :ref:`event_mail`)                                                   |
-+---------------------+------------------------------------------------------+
+   Inherits from fs or any other specified event (e.g. mail).
 
-.. _event_fs_iter:
+   .. todo:: WHAT DOES THIS MEAN???
 
-fs_file
--------
 
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| Inherits from :ref:`event_fs` or any other specified event                 |
-| (e.g. :ref:`event_mailbox`)                                                |
-+---------------------+------------------------------------------------------+
-
-If the file was created for obox, it has also fields:
-
-+---------------------+------------------------------------------------------+
-| Field               | Description                                          |
-+=====================+======================================================+
-| file_type           | * mail: Email file                                   |
-|                     | * index: Index bundle                                |
-|                     | * box: Mailbox directory (for creating/deleting it,  |
-|                     |   if used by the storage driver)                     |
-|                     | * fts: FTS file                                      |
-+---------------------+------------------------------------------------------+
-| reason              | Reason for accessing the file                        |
-+---------------------+------------------------------------------------------+
+.. dovecot_core:event:: fs_iter
+
+   :field file_type: * ``mail``: Email file
+                     * ``index``: Index bundle
+                     * ``box``: Mailbox directory (for creating/deleting it,
+                       if used by the storage driver)
+                     * ``fts``: FTS file
+   :field reason: Reason for accessing the file.
+
+   Inherits from fs or any other specified event (e.g. mailbox).
+
+   If the file was created for obox, it also has fields:
```

Comment:
Hmm. Something's wrong here now. The "file_type" and "reason" are supposed to be explained below this, they don't exist for all fs_files. And this is supposed to be for fs_file event, not for fs_iter event. Looks like that was wrong already in the original page.